meerk40t/gui/scene/scene.py

```diff
@@ -142,6 +145,44 @@ def signal(self, *args, **kwargs):
         """
         self.scene._signal_widget(self.scene.widget_root, *args, **kwargs)
 
+    # Indicator for Keyboard-Modifier
```

Comment:
Sorry - but IMO you need to use get_key_name in wxutils.py to check modifiers. This is the single place in Meerk40t that we sort out modifiers in a way that will be consistent across Windows, Linux and Mac.

@tatarize gets the final say, but personally I think this is an essential change.